src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -0,0 +1,663 @@
+/**
+ * @brief Read-only property to get a std::string value representing a full device name.
+ */
+static constexpr Property<std::string, PropertyMutability::RO> full_name{"FULL_DEVICE_NAME"};
+
+/**
+ * @brief Read-only property which defines the device architecture.
+ */
+static constexpr Property<std::string, PropertyMutability::RO> architecture{"DEVICE_ARCHITECTURE"};
+
+/**
+ * @brief Enum to define possible device types
+ */
+enum class Type {
+    INTEGRATED = 0,  //!<  Device is integrated into host system
+    DISCRETE = 1,    //!<  Device is not integrated into host system
+};
+
+/** @cond INTERNAL */
+inline std::ostream& operator<<(std::ostream& os, const Type& device_type) {
+    switch (device_type) {
+    case Type::DISCRETE:
+        return os << "discrete";
+    case Type::INTEGRATED:
+        return os << "integrated";
+    default:
+        throw ov::Exception{"Unsupported device type"};
+    }
+}
+
+inline std::istream& operator>>(std::istream& is, Type& device_type) {
+    std::string str;
+    is >> str;
+    if (str == "discrete") {
+        device_type = Type::DISCRETE;
+    } else if (str == "integrated") {
+        device_type = Type::INTEGRATED;
+    } else {
+        throw ov::Exception{"Unsupported device type: " + str};
+    }
+    return is;
+}
+/** @endcond */
+
+/**
+ * @brief Read-only property to get a type of device. See Type enum definition for possible return values
+ */
+static constexpr Property<Type, PropertyMutability::RO> type{"DEVICE_TYPE"};
+
+/**
+ * @brief Read-only property which defines Giga OPS per second count (GFLOPS or GIOPS) for a set of precisions supported
+ * by specified device
+ */
+static constexpr Property<std::map<element::Type, float>, PropertyMutability::RO> gops{"DEVICE_GOPS"};
+
+/**
+ * @brief  Read-only property to get a float of device thermal
+ */
+static constexpr Property<float, PropertyMutability::RO> thermal{"DEVICE_THERMAL"};
+
+/**
+ * @brief Read-only property to get a std::vector<std::string> of capabilities options per device.
+ */
+static constexpr Property<std::vector<std::string>, PropertyMutability::RO> capabilities{"OPTIMIZATION_CAPABILITIES"};
+namespace capability {
+constexpr static const auto FP32 = "FP32";                    //!< Device supports fp32 inference
+constexpr static const auto BF16 = "BF16";                    //!< Device supports bf16 inference
+constexpr static const auto FP16 = "FP16";                    //!< Device supports fp16 inference
+constexpr static const auto INT8 = "INT8";                    //!< Device supports int8 inference
+constexpr static const auto BIN = "BIN";                      //!< Device supports binary inference
+constexpr static const auto WINOGRAD = "WINOGRAD";            //!< Device supports winograd optimization
+constexpr static const auto EXPORT_IMPORT = "EXPORT_IMPORT";  //!< Device supports model export and import
+constexpr static const auto BATCHED_BLOB = "BATCHED_BLOB";    //!< Device supports blobs batching
```

Comment:
`BATCHED_BLOB `should be removed from new API

Reply:
removed